tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -159,10 +158,47 @@ func simulateTraffic(ctx context.Context, hosts []string, ids identity.Provider,
 		reports = append(reports, c.Report())
 		mux.Unlock()
 	}(compactClient)
+	dd := time.Duration(robustnessrand.RandRange(3, 37)) * traffic.DefaultCompactionPeriod
+	for _, h := range hosts {
+		c := connect([]string{h}, ids, baseTime)
+		wg.Add(1)
+		go func(c *client.RecordingClient, lg *zap.Logger) {
+			defer wg.Done()
+			defer c.Close()
+			runDefragLoop(ctx, lg, c, dd, finish)
+			mux.Lock()
+			reports = append(reports, c.Report())
+			mux.Unlock()
+		}(c, lg)
+	}
 	wg.Wait()
 	return reports
 }
 
+func runDefragLoop(ctx context.Context, lg *zap.Logger, c *client.RecordingClient, period time.Duration, finish <-chan struct{}) {
+	for {
+		jitter := robustnessrand.RandRange(int64(period-period/2), int64(period+period/2))
+		nextPeriod := period + time.Duration(jitter)
```

Comment:
jitter variable is from `period/2` to `period * 3/2` this makes sense assuming jitter 0.5, then why the  `nextPeriod` adds another `period`?